projects/sbb-esta/angular-public/src/lib/tooltip/tooltip/tooltip.component.ts

```diff
@@ -334,4 +347,39 @@ export class TooltipComponent implements OnDestroy {
       }
     });
   }
+
+  /**
+   * onMouseOver method used to show the tooltip when trigger type is set to 'hover'
+   * and the mouse hover the button
+   */
+  onMouseOver($event: MouseEvent) {
+    this.clearTimoutOnChangeMouseEvent();
+    event.stopPropagation();
+    this._referenceTimoutMouseOver = setTimeout(() => {
```

Comment:
Prefer inline lambdas, where there is only one command
e.g. `setTimeout(() => this.open(true), this.hoverOpenDelay);`